install.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# CHECKS FOR SUDO ACCESS (added because of those who have seperate partitions for /home and /root) 
+
+if [ "$EUID" -ne 0 ]
```

Comment:
This should say -eq, right now it's only allowing operation as root

Reply:
OH

Reply:
fixed it